newdle/api.py

```diff
@@ -292,6 +293,8 @@ def update_newdle(args, code):
         raise Forbidden
     for key, value in args.items():
         setattr(newdle, key, value)
+    if newdle.final_dt:
+        newdle.last_update = datetime.utcnow()
```

Comment:
What about adding a method like `newdle.update_lastmod()` and call that whenever something is changed that needs to bump it?